ComfortZoneApp/app/src/main/java/com/example/comfortzone/HostActivity.java

```diff
@@ -67,7 +73,8 @@ private void maybeUpdateComfortLevel() {
         ParseUser currentUser = ParseUser.getCurrentUser();
         ArrayList<ComfortLevelEntry> todayEntries = (ArrayList<ComfortLevelEntry>) currentUser.get(ComfortLevelUtil.KEY_TODAY_ENTRIES);
         if (!todayEntries.isEmpty() && todayEntries.get(0).getUpdatedAt() != null && !DateUtils.isToday(todayEntries.get(0).getUpdatedAt().getTime())) {
-            ComfortLevelUtil.updateComfortLevel(currentUser);
+            updateComfortLevel(currentUser);
```

Comment:
@fionamei for the util class, instead of importing the functions import static com.example.comfortzone.utils.ComfortCalcUtil.calculateAverages you can just do an import of the class com.example.comfortzone.utils.ComfortCalcUtil and call the function as ComfortCalcUtil.updateComfortLevel(currentUser). Otherwise it gives an impression that updateComfortLevel(currentUser) function is defined int he same class HostActivity.